src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -48,4 +48,10 @@ public Order(String status) {
         dateCreated = new Date();
         this.status = status;
     }
+
+    public Order(Long id, Date date, String status) {
+        this.id = id;
+        this.dateCreated = date;
+        this.status = status;
+    }
```

Comment:
Tutaj to samo z ID. Ja bym nie dawał do konstruktora pola Id

Reply:
Tez mi sie wydaje ,ze nie potrzeba tutaj dodawac pola ID, bo jest generowane automatycznie.